CRM/Civioffice/DocumentRendererType/LocalUnoconv.php

```diff
@@ -121,7 +121,8 @@ public function isReady(): bool
                 return false;
             }
         } catch (Exception $ex) {
-            Civi::log()->debug("CiviOffice: Unoconv generic exception in isReady() check");
+            CRM_Core_Session::setStatus($ex->getMessage(), E::ts('CiviOffice Error'), 'error');
```

Comment:
Not sure we want to expose the actual exception message to the UI. Maybe something like *Unknown error, see the error log for details.* …?

Reply:
I'm not married to my suggestion, but displaying raw exception messages in the UI doesn't feel right …